docs/sources/configure/options.md

```diff
@@ -753,25 +790,19 @@ attacks. This option should be used only for testing and development purposes.
 
 Configures the intervening time between exports.
 
-| YAML            | Environment variable                       | Type    | Default |
-| --------------- | ----------------------------- | ------- | ------- |
-| `report_target` | `BEYLA_METRICS_REPORT_TARGET` | boolean | `false` |
```

Comment:
`report_target` and `report_peer` are still supported but they became deprecate with the new attributes selection feature. That's why I removed the documentation.